transports/small-webrtc-transport/src/smallWebRTCTransport.ts

```diff
@@ -221,12 +267,29 @@ export class SmallWebRTCTransport extends Transport {
 
     this.state = "connecting";
 
-    this._webrtcUrl =
-      connectParams?.webrtcUrl ??
-      connectParams?.connectionUrl ??
-      this._webrtcUrl;
-    if (!this._webrtcUrl) {
-      logger.error("No url provided for connection");
+    const _webrtcUrl =
```

Comment:
This logic seems to duplicate what we already do in the constructor. Could we unify it ? (checking webrtcUrl/connectionUrl, printing warnings, creating _webrtcRequest)

Reply:
nm. i'm doing it.